sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_differentiate_custom_forms_with_labelled_and_unlabelled_models_async.py

```diff
@@ -0,0 +1,95 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_differentiate_custom_forms_with_labeled_and_unlabeled_models_async.py
+
+DESCRIPTION:
+    This sample demonstrates the differences in output that arise when recognize_custom_forms
+    is called with custom models trained with labeled and unlabeled data.
+USAGE:
+    python sample_differentiate_custom_forms_with_labeled_and_unlabeled_models_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+    3) CUSTOM_TRAINED_LABELED_MODEL_ID - the ID of your custom model trained with labeled data
+    4) CUSTOM_TRAINED_UNLABELED_MODEL_ID - the ID of your custom model trained with unlabeled data
+"""
+
+import os
+import asyncio
+
+
+class DifferentiateCustomFormsWithlabeledAndUnlabeledModelsAsync(object):
```

Comment:
Do we want to consider grouping the advanced samples in a separate folder so that the user is not too overwhelmed looking at these in the general samples folder?

Reply:
I see where you're coming from. I don't know about other examples of this, I'll bring it up during scrum to see where people stand. Thanks!